mb_modelbase/server/modelbase.py

```diff
@@ -267,6 +273,15 @@ def list_models(self):
         return list(self.models.keys())
 
     def execute(self, query):
+        path = ""
+        queryLogName = "interaction" + \
+                       time.strftime("%b:%d:%Y_%H", time.gmtime(time.time())) + ".log"
+
+        if not "SHOW" in query.keys():
+            with open(queryLogName, "a") as f:
+                f.write(json.dumps(query) + '\n')
+                logger.info(json.dumps(query))
```

Comment:
Is there a way to disable logging? it is not always necessary, as it really is just logging, correct? 
Please add a way to disable it, and make it the default.

Reply:
Done!